DESCRIPTION

```diff
@@ -10,7 +10,8 @@ Description: Run 'R CMD check' from 'R' and capture the results
     of the individual checks. Supports running checks in the background,
     timeouts, pretty printing and comparing check results.
 License: MIT + file LICENSE
-URL: https://r-lib.github.io/rcmdcheck/, https://github.com/r-Lib/rcmdcheck#readme
+URL: https://rcmdcheck.r-lib.org/,
+    https://github.com/r-Lib/rcmdcheck#readme
```

Comment:
```suggestion
    https://github.com/r-lib/rcmdcheck#readme
```

This is just a weird little thing I noticed last night, actually, when I was looking at all these.